packages/plugins/plugin-sheet/src/SheetPlugin.tsx

```diff
@@ -138,6 +132,25 @@ export const SheetPlugin = (): PluginDefinition<SheetPluginProvides> => {
           },
         ],
       },
+      thread: {
+        predicate: (data) => data instanceof SheetType,
+        createSort: (sheet) => {
+          return (anchorA, anchorB) => {
```

Comment:
move generally stateless and abstract functions like out of scope of plugin and with TODO to factor out into util.